lib/matplotlib/backends/_backend_gtk.py

```diff
@@ -109,6 +112,143 @@ def _on_timer(self):
             return False
 
 
+class _FigureManagerGTK(FigureManagerBase):
+    """
+    Attributes
+    ----------
+    canvas : `FigureCanvas`
+        The FigureCanvas instance
+    num : int or str
+        The Figure number
+    toolbar : Gtk.Toolbar
```

Comment:
No GtkToolbar in GTK4:
```suggestion
    toolbar : Gtk.Toolbar or Gtk.Box
```

Reply:
indeed